src/test/java/org/fedoraproject/javapackages/validator/validators/RpmFilesizeValidatorTest.java

```diff
@@ -0,0 +1,35 @@
+package org.fedoraproject.javapackages.validator.validators;
+
+public class RpmFilesizeValidatorTest {
```

Comment:
Test is not implemented.
If the test compiles but fails then it would be better to use [Disabled annotation of JUnit](https://junit.org/junit5/docs/current/user-guide/#writing-tests-disabling).